macros/models/_sector/transfers/transfers_native.sql

```diff
@@ -0,0 +1,182 @@
+{% macro transfers_native(blockchain, traces, transactions, native_token_address, genesis_balances=null, native_erc_transfers=null) %}
+
+WITH
+
+native_transfers  as (
+        SELECT 
+            'receive' as transfer_type, 
+            tx_hash,
+            trace_address, 
+            block_time,
+            to as wallet_address, 
+            {{native_token_address}} as token_address,
```

Comment:
why is there no "from" here?

How will I know what addresses the transfer is between?